src/js/webrequest.js

```diff
@@ -103,6 +103,7 @@ function onBeforeRequest(details){
       };
       chrome.tabs.sendMessage(tab_id, msg);
 
+      badger.checkForDNTPolicy(requestDomain, badger.storage.getNextUpdateForDomain(requestDomain));
```

Comment:
Should we make both checkForDNTPolicy calls in webrequest.js async, to avoid delaying request resolution as much as we can? Previously requested in https://github.com/EFForg/privacybadger/pull/1642#discussion_r137572271.

Reply:
done